target/linux/ar71xx/image/generic.mk

```diff
@@ -196,6 +209,45 @@ define Device/mr16
 endef
 TARGET_DEVICES += mr16
 
+define Device/mr1750
+  DEVICE_TITLE := OpenMesh MR1750
+  DEVICE_PACKAGES := om-watchdog kmod-ath10k ath10k-firmware-qca988x
+  BOARDNAME = MR1750
+  FILESYSTEMS := squashfs
+  BLOCKSIZE := 4 8 16 64 128 256
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image board=MR1750 rootfs=$$$$@
```

Comment:
```IMAGE/sysupgrade.bin := ``` should be enough. Same for the factory.bin.

I'm not yet sure what combined-ext-image really does but I'm not that happy that the rootfs is passed by default. Similar is possible with sysupgrade-tar but it is only used for some special cases. sysupgrade-tar without extra param fits in most cases.

It should be possible to use $(BOARDNAME) within the combined-ext-image recipe instead of passing it for each OpenMesh image. Check the existing image build code in include/image-commands.mk for some examples. You can make it even conditional.